app/ui-react/packages/ui/src/Data/Virtualizations/PublishStatusWithProgress.tsx

```diff
@@ -34,17 +36,25 @@ export const PublishStatusWithProgress: React.FunctionComponent<
           }
           i18nLogUrlText={props.i18nPublishLogUrlText}
         />
+        {props.publishVersion
```

Comment:
This should be i18n'd but can be worked in a follow-on issue. Also in a future issue, like we talked about, using an 'M' as modified indicator needs to be changed per UX guidance/mockups. There are multiple occurrences that need to be changed.